ingestion/values.yaml

```diff
@@ -0,0 +1,14 @@
+replicaCount: 1
+
+image:
+  repository: nginx
```

Comment:
I think the image repository here is leftover from the generated file that `kubernetes create chart` makes. The repository should probably be our Github container registry, not nginx. (I noticed it was also an issue on the record linkage service, we should change it there too)